### PR TITLE
CLOUDP-70060: mongocli ops-manager|cloud-manager backups config describe 

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-70060

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli om backup config list -P ops -o json
{
  "links": [
    {
      "rel": "self",
      "href": "http://localhost:8080/api/public/v1.0/groups/5f6cc16511c3aa3a90b5ec34/backupConfigs?pageNum=1\u0026itemsPerPage=100"
    }
  ],
  "results": [
    {
      "groupId": "5f6cc16511c3aa3a90b5ec34",
      "clusterId": "5f6dfb3a11c3aa3a90b66db6",
      "statusName": "INACTIVE"
    },
    {
      "groupId": "5f6cc16511c3aa3a90b5ec34",
      "clusterId": "5f719b052845cb05726a860b",
      "statusName": "STARTED",
      "storageEngineName": "WIRED_TIGER"
    },
    {
      "groupId": "5f6cc16511c3aa3a90b5ec34",
      "clusterId": "5f719b352845cb05726a86f9",
      "statusName": "STARTED",
      "storageEngineName": "WIRED_TIGER"
    }
  ],
  "totalCount": 3
}
```
```

./bin/mongocli om backup config list -P ops
PROJECT ID                 CLUSTER ID                 STATUS     STORAGE ENGINE
5f6cc16511c3aa3a90b5ec34   5f6dfb3a11c3aa3a90b66db6   INACTIVE
5f6cc16511c3aa3a90b5ec34   5f719b052845cb05726a860b   STARTED    WIRED_TIGER
5f6cc16511c3aa3a90b5ec34   5f719b352845cb05726a86f9   STARTED    WIRED_TIGER
```

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
waiting for https://github.com/mongodb/go-client-mongodb-ops-manager/pull/55

File: internal/cli/opsmanager/backup/config/describe.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli ops-manager backup config describe clusterID [--projectId projectId]
+func DescribeBuilder() *cobra.Command {
+	opts := &DescribeOpts{}
+	cmd := &cobra.Command{
+		Use:     "describe [clusterID]",
```
Comment (gssbzn) [inline internal/cli/opsmanager/backup/config/describe.go:56]: [nit] brackets ([]) for optional, for mandatory the standard is less than, greater than (<>)
```suggestion
		Use:     "describe <clusterID>",
```

**Review by gssbzn — CHANGES_REQUESTED** [review]
Just one small nit

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**